include/wx/xml/xml.h

```diff
@@ -222,6 +222,16 @@ enum wxXmlDocumentLoadFlag
     wxXMLDOC_KEEP_WHITESPACE_NODES = 1
 };
 
+// Create an instance of this and pass it to wxXmlDocument::Load()
+// to get detailed error information in case of failure. Values
+// are set to -1 if no error occurred.
+struct wxXmlParseError
+{
+    wxString message;
+    int line;
+    int column;
+    int byte_offset;
```

Comment:
I think this one should use `wxFileOffset` as it's conceivable to work with the files > 2GiB.

If not, the casts to it will at need to be checked.